network/load-balancer/how-to/add-certificate.mdx

```diff
@@ -199,11 +197,10 @@ View the certificate(s) that you have uploaded for your Load Balancer on the **S
 Use the <Icon name="delete" /> button to delete unwanted certificates.
 
 <Message type="tip">
-
 If you have more than one certificate for your Load Balancer, it will automatically use the first in the list, regardless of its expiry date. Ensure that you delete any expired certificates, and that the current certificate you want it to use is first in the list.
```

Comment:
```suggestion
If you have more than one certificate for your Load Balancer, it will automatically use the first in the list, regardless of its expiry date. Ensure that you delete any expired certificates, and that the current certificate you want to use is first in the list.
```